python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -257,60 +256,87 @@ def repackage_eimage(self, phosim_eimg_file, out_dir='.'):
             PhoSim eimage file.
         out_dir : str, optional
             Output directory (the default is '.'.)
+        verbose: bool, optional
+            Set to True to print out processing information.
         """
+        TELCODE = self.telcode
+        CONTRLLR = self.contrllr
```

Comment:
Same questions above.

Reply:
As above - changed to `self.TELCODE` throughout